packages/plugin-ext/src/common/language-pack-service.ts

```diff
@@ -0,0 +1,34 @@
+// *****************************************************************************
+// Copyright (C) 2023 TypeFox and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0.
+//
+// This Source Code may also be made available under the following Secondary
+// Licenses when the conditions for such availability set forth in the Eclipse
+// Public License v. 2.0 are satisfied: GNU General Public License, version 2
+// with the GNU Classpath Exception which is available at
+// https://www.gnu.org/software/classpath/license.html.
+//
+// SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+// *****************************************************************************
+
+/**
+ * Starting with vscode 1.73.0, language pack bundles have changed their shape to accomodate the new `l10n` API.
+ * They are now a record of { [englishValue]: translation }
```

Comment:
Yep, but that's the direction VSCode is heading. The `l10n.t` API only accepts an english default text, which gets exchanged with a localized version in case we're not using `en` as our locale.